doc/21-development.md

```diff
@@ -1357,81 +1354,9 @@ mkdir -p release debug
 Proceed with the specific distribution examples below. Keep in mind that these instructions
 are best effort and sometimes out-of-date. Git Master may contain updates.
 
-* [CentOS 7](21-development.md#development-linux-dev-env-centos)
```

Comment:
By omitting this section completely, we no longer have dev setup docs for RPM distros. So instead of dropping this section entirely, please modify the instructions to work on either `RHEL` or `Fedora` systems.